openreview/conference/builder.py

```diff
@@ -1233,7 +1241,7 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
 
         If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
 
-        If you have any questions, please contact info@openreview.net.
+        If you have any questions, please contact ''' + contact_email + '''.
```

Comment:
Please add `{contact_info}` as part of the template and add the contact_info parameter in `tools.recruit_reviewers`.